schema/tracer_provider.json

```diff
@@ -30,5 +42,80 @@
                 }
             }
         }
+    },
+    "$defs": {
+        "SimpleSpanProcessor": {
+            "type": "object",
+            "additionalProperties": false,
+            "title": "SimpleSpanProcessor",
+            "properties": {
+                "exporter": {
+                    "$ref": "#/$defs/SpanExporter"
+                }
+            }
+        },
+        "BatchSpanProcessor": {
+            "type": "object",
+            "additionalProperties": false,
+            "title": "BatchSpanProcessor",
+            "properties": {
+                "exporter": {
+                    "$ref": "#/$defs/SpanExporter"
+                }
+            }
+        },
+        "SpanExporter": {
+            "type": "object",
+            "additionalProperties": true,
+            "patternProperties": {
+                "^otlp.*": {
+                    "$ref": "#/$defs/Otlp"
+                },
+                "^console.*": {
+                    "$ref": "#/$defs/Console"
+                }
+            }
+        },
+        "Console": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {}
+        },
+        "Otlp": {
```

Comment:
Most of this looks in common with the metric OTLP exporter. Not super familiar with JSON schema but it could be nice to reuse some of it so they don't go out of sync

Reply:
There are only 2 fields which are different, i suppose since those fields are not required, they could be the same definition... only downside i could think of is someone getting prompted to fill in unnecessary fields for a span/log exporter

Reply:
I've combined the otlp + console exporters under common.json, PTAL